packages/advisor-components/src/RuleDetails/RuleDetails.spec.ct.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
This is a single spec I am adding within this patch. Serves more like an illustration of typical cypress specs and tests RuleDetails component.